### PR TITLE
chart_legend

### PR DESCRIPTION
**Pull request by punx120**
I know the legend is on the left side - but i often copy the chart and paste it in an email or chat - and in that case the legend is absent which is annoying

THE FOLLOWING DISCLAIMER APPLIES TO ALL SOFTWARE CODE AND OTHER MATERIALS CONTRIBUTED IN CONNECTION WITH THIS SOFTWARE:
THIS SOFTWARE IS LICENSED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS “AS IS” AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE AND ANY WARRANTY OF NON-INFRINGEMENT, ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE. THIS SOFTWARE MAY BE REDISTRIBUTED TO OTHERS ONLY BY EFFECTIVELY USING THIS OR ANOTHER EQUIVALENT DISCLAIMER IN ADDITION TO ANY OTHER REQUIRED LICENSE TERMS.
ONLY THE SOFTWARE CODE AND OTHER MATERIALS CONTRIBUTED IN CONNECTION WITH THIS SOFTWARE, IF ANY, THAT ARE ATTACHED TO (OR OTHERWISE ACCOMPANY) THIS SUBMISSION (AND ORDINARY COURSE CONTRIBUTIONS OF FUTURES PATCHES THERETO) ARE TO BE CONSIDERED A CONTRIBUTION. NO OTHER SOFTWARE CODE OR MATERIALS ARE A CONTRIBUTION.

### CODE REVIEW

File: src/main/studio/ui/chart/Chart.java [hunk 1]
```diff
@@ -270,7 +270,7 @@ private JFreeChart createChart() {
         if (xAxis == null) return null;
 
         JFreeChart chart = new JFreeChart("", JFreeChart.DEFAULT_TITLE_FONT,
-                plot, false);
+                plot, true);
```
Comment (punx120) [inline src/main/studio/ui/chart/Chart.java:273]: This is the main change - everything else is _style_

**Comment by dzmipt** [comment]
I intentionally removed legends as they can take a lot of space in case of many columns + long column names. Technically legends are on the right panel. Although now I understand that on copying or saving the chart, legend can be useful. I can propose to have an option (a check box) which show/hide the legend.

**Comment by punx120** [comment]
> I intentionally removed legends as they can take a lot of space in case of many columns + long column names. Technically legends are on the right panel. Although now I understand that on copying or saving the chart, legend can be useful. I can propose to have an option (a check box) which show/hide the legend.

Sure - I can add that

**Comment by punx120** [comment]
Done - added a check box

**Comment by dzmipt** [comment]
> Done - added a check box

Hmm... i can't find this change (I did not work a lot with pull-request flow in the github... may be I don't know where to find new changes?)

**Comment by gyorokpeter** [comment]
@punx120 Did you forget to push?

**Comment by punx120** [comment]
Yeah i think due to some rebase - udpate didn't flow in automatically opened a new PR #96

**Pull request closed without merging**